public/app/features/alerting/unified/components/import-to-gma/ImportFromDSRules.tsx

```diff
@@ -38,11 +42,22 @@ export interface ImportFormValues {
   targetDatasourceUID?: string;
```

Comment:
I think we could make more clear what this target datasource is? , maybe with a comment, maybe changing the naming? like tagertDataSourceForRecordingRules?